jsk_panda_robot/jsk_panda_startup/launch/dual_panda2.launch

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" ?>
+<launch>
+
+  <arg name="rarm_ip" default="172.16.0.10"/>
+  <arg name="larm_ip" default="172.16.0.100"/>
```

Comment:
One idea is to setup same two IPs for two dual-arm robot system, that would be easier to switch your program from one robot system to another.
if you want to setup four different IP for all individual panda robot (that's would be nice if we want to build 4-armed robot), In that case, how about use PANDA_RARM_IP/PANDA1_IP environment variable like NAO_IP and use same launch file.

Or at least, we have `dual_panda.xml` and `dual_panda1.launch` and `dual_panda2.launch` should include taht file.

Reply:
> One idea is to setup same two IPs for two dual-arm robot system, that would be easier to switch your program from one robot system to another.

This seems realizable by introducing one more Ethernet switch to `dual_panda2` and creating another `172.16.*` network.
I know nothing about panda network setting, so I'll study it and try.

> Or at least, we have dual_panda.xml and dual_panda1.launch and dual_panda2.launch should include taht file.

I didn't create `dual_panda.xml` because I thought common part of `dual_panda.launch` and `dual_panda2.launch` is already almost one launch file.
That common part is here:
https://github.com/pazeshun/jsk_robot/blob/bea7e509a1a027169ea06f4df74fe908e8a41443/jsk_panda_robot/jsk_panda_startup/launch/dual_panda.launch#L25-L49
`franka_combined_control.launch` is the common launch file, which is in `franka_control`.
The common part also includes `franka_gripper.launch`, but I thought this may become not common if we create another dual_panda which has other grippers.
If you still think we should create `dual_panda.xml` in spite of this circumstances, OK, I'll create it.

Current `dual_panda` and `dual_panda2` are different in attached devices:
- `dual_panda`
  - Xtion camera on head (torso?)
  - Web cameras on both hands
  - Microphone on right hand
- `dual_panda2`
  - RealSense D435 on both hands

Perhaps do you intend to create many arguments in the common launch file like https://github.com/jsk-ros-pkg/jsk_robot/blob/b40df2f1086bfd8b0c76a83284a7a3a5ad9b85f6/jsk_baxter_robot/jsk_baxter_startup/baxter.launch#L2-L31?
I unconsciously avoided this style because this is hard to read for me, but this is just my preference, so I'll follow you if you intend that style.

By the way, I kept the name of the first robot `dual_panda` because I was afraid to break @ykawamura96 's code, but changing to `dual_panda1` is better?

Reply:
if you have common part, do not re-write them, but re-use it.
If you do not like to add many `args`, you can  create common files `dual_panda.xml` without args, and include this from `dual_panda1.launch` and `dual_panda2.launch` and put for example RealSense parts on `dual_panda2.launch`

Reply:
I think dual_panda1 is better, 
if any can run @ykawamura96 's code, please change that code. if no one can run his code, then changing launch file name does not break anything... unfortunately.

Reply:
> if you have common part, do not re-write them, but re-use it.

My point was that truly common part is already one launch file in `franka_control` and we re-used it.
However, now I understand that some argument settings should be common.
Also, maybe you mean `franka_gripper.launch` should be common.
I'll create common launch.

Reply:
> if any can run @ykawamura96 's code, please change that code. if no one can run his code, then changing launch file name does not break anything... unfortunately.

Unfortunately, no one can run his code now.
But as I wrote in https://github.com/jsk-ros-pkg/jsk_robot/pull/1544#discussion_r929668531, I have to learn his system, so maybe I'll check his code can run even after robot name is changed.

Reply:
> If you do not like to add many `args`, you can create common files `dual_panda.xml` without args, and include this from `dual_panda1.launch` and `dual_panda2.launch` and put for example RealSense parts on `dual_panda2.launch`

After this is done, currently I'm planning to follow

> One idea is to setup same two IPs for two dual-arm robot system, that would be easier to switch your program from one robot system to another.

- Advantage
  - If we purchase another dual_panda, we can test this by `dual_panda.xml` with the same network configuration as the existing robots
- Disadvantage
  - Taking some time
  - Using one more Ethernet switch

Please stop me if I go to a wrong way.

Reply:
Done